CHANGELOG

```diff
@@ -11,10 +11,12 @@ CHANGES
 -----
 CONTRIBUTORS
 - JeS24
+- mshumayl
 
 CHANGES
 - [#607]: Fixed #586 #598 #599 #600 #601, Fixed a bug in tensor_lambdify
 - [#609]: Fixed #608, Fixed notebook preview errors in documentation
+- [#612]: Partially addressed #610, Added support for setting plot aspect ratio and title via API
```

Comment:
Maintain chronological order for PRs here rather than numerical order. So, this should be:
```
- [#607]: Fixed #586 #598 #599 #600 #601, Fixed a bug in tensor_lambdify
- [#609]: Fixed #608, Fixed notebook preview errors in documentation
- [#615]: Partially fixed #613, Updated README and CONTRIBUTING
- [#617]: Fixed #616, Dropped support for Python 3.7, Added support for Python 3.10
- [#612]: Partially addressed #610, Added support for setting plot aspect ratio and title via API
```